app/scripts/lib/crypto/scoped-keys.js

```diff
@@ -6,20 +6,18 @@
  * Derive scope-specific keys from account master keys.
  */
 
-
-function importFxaCryptoDeriver() {
-  return import(/* webpackChunkName: "fxaCryptoDeriver" */ 'fxaCryptoDeriver');
-}
+import ecdh from './ecdh';
```

Comment:
Can probably revert this change since Desktop takes care of creating the final keys_jwe and ecdh is no longer used.